src/seedu/addressbook/commands/Command.java

```diff
@@ -46,6 +46,15 @@ public CommandResult execute(){
     //Note: it is better to make the execute() method abstract, by replacing the above method with the line below:
     //public abstract CommandResult execute();
 
+    /**
+     *  Checks if command type mutates the data
```

Comment:
A better header command in this case will just be returns true if the command ...... To further improve on it, you can think about whether it can also return true if the command may not be mutating. A exception would be adding a duplicate person to the unique list. In this case, nothing will be added. Therefore, phrasing it as "potentially mutate" would be better.